public/js/buildGherkin.js

```diff
@@ -310,7 +314,7 @@ function buildTable(testEl) { // el should be the test element
             if ($(this).hasClass('ui-autocomplete-input')) {
                 return true;
             }
-            if ($(this).val() == "" && $(this).is("input")) {
+            if ($(this).val() === "" && $(this).is("input")) {
                 var placeholder = $(this).attr('placeholder')
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "placeholder" as this name is already used in declaration at line 305. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=javascript%3AS2814)